server/src/modules/search/__tests__/search.service.spec.ts

```diff
@@ -0,0 +1,247 @@
+import { StatusCodes } from 'http-status-codes'
+import { SearchEntry, SearchService } from '../search.service'
+import { Mocker } from './opensearch-mock'
+// import { baseConfig, Environment } from '../../../bootstrap/config/base'
+// import { Client, errors, Connection } from '@opensearch-project/opensearch'
+// import fs from 'fs'
+
+const { Client, errors } = require('@opensearch-project/opensearch') // eslint-disable-line
```

Comment:
```suggestion
// eslint-disable-next-line @typescript-eslint/no-require-imports
const { Client, errors } = require('@opensearch-project/opensearch')
```

Reply:
Where did you find the documentation to include the `@typescript-eslint/no-require-imports` part? It's complaining :/

<img width="948" alt="Screenshot 2021-11-10 at 11 08 11 AM" src="https://user-images.githubusercontent.com/37061143/141043701-61645b1e-e814-4a69-a8bf-7063e7367fa0.png">

Reply:
nevermind, it was the wrong rule. it's fine now